listenbrainz/labs_api/labs/api/recording_from_recording_mbid.py

```diff
@@ -54,45 +56,84 @@ def fetch(self, params, offset=-1, count=-1):
                     r = dict(row)
                     redirect_index[str(r['recording_mbid_old'])] = str(r['recording_mbid_new'])
```

Comment:
No strong opinions either way, but in AB when we deal with uuids we often `SELECT x::text` in the sql query to avoid having to call `str` everywhere in python

Reply:
I like that suggestion!